### DISCUSSION TITLE
Who wants to be hired (Aug 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#39274] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #38459. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #39273.

**Comment by amirardalan** [comment]
I'm a Front-end Developer with 10+ years experience building outstanding digital products for top brands. I'm currently open to contract or full-time React/Next.js work.

I possess extensive design experience in addition to full stack engineering skills. I'm especially passionate about User Interface and Front-end. I lean toward functional programming paradigms and build my React projects using TypeScript and hooks. I eat CSS for breakfast.

Reach out if you're interested in working together!

Location: Portland, OR, USA
Remote: Yes
Willing to relocate: No
Technologies: TypeScript, React, Next.js, Emotion, Adobe XD, SWR, PostgreSQL, Vercel
Resume/CV: https://amirardalan.com/amir-ardalan-resume.pdf
Website: https://amirardalan.com/
Github: https://github.com/amirardalan
Email: [hi@amirardalan.com](mailto:hi@amirardalan.com)

**Comment by Fasunle** [comment]
Hi,

I am a fullstack developer with 3 years of work experience in MERN stack + Python Development. I am currently open full-time full stack work.

Location: Nigeria
Remote: Yes
Willing to relocate: Yes
Technologies: React, ExpressJS, css, html5, MongoDB, PostgreSQL, TypeScript, Python, Flask, GraphQL, Redux, Docker, Kubernetes, CI/CD workflow with AWS, RESTful API development and integration.
Resume/CV: [Uploading KehindeFasunleFullstackDeveloper.pdf…] ()
Website: None
GitHub: https://github.com/Fasunle
Email: fasunlekehinde11@gmail.com

**Comment by albertkimdev** [comment]
SEEKING WORK | REMOTE (Thailand time) 

Available for part time freelance work.

I'm a frontend developer specializing in Next.js and Javascript web applications.

I started my first internship in 2014, and have been freelancing since 2016, which is when I started learning React and Node.

I created an NFT sports card marketplace for high school athletes - https://verifiedink.us/ - I worked with my client from the beginning of the project, gathering requirements, architecting the project, and continue to work on the features today.

It's built with Next.js, Supabase, Chakra UI, Stripe, & Solana.

My own project is a no-code website portfolio builder for freelancers built with Next.js, Styled Components, Node/Express, Stripe, and more. - https://leavewithaweb.site/

It's been through dozens of iterations from the code base to the UI/UX and more over the past 2 years.

I am currently interested in Next.js full-stack development work or anything frontend coding related.

I will be based in Thailand for the next few months at least.

Rate - $70us / hr

website: https://albertkim.co/

Github: https://github.com/albertkimdev

**Comment by thinktapper** [comment]
**ʰᵉʸᵒ** ✌️ I'm **Tapper**, a highly motivated and flexible Software Engineer with a passion for identifying creative solutions to complex problems. Data-driven, dependable, and enthusiastic about working with new and exciting languages & technologies. I possess an exceptional aptitude in managing multiple projects and tackling new tasks as priorities shift.

Location: **Minneapolis, Minnesota USA**
Remote: **Yᗩᔕᔕ**
Willing to relocate: **Yᗩᔕᔕ**
Technologies: **Next.js ⟡ React ⟡ GraphQL ⟡ Prisma ⟡ MongoDB ⟡ PostgreSQL ⟡ Shopify ⟡ Sass ⟡ TailwindCSS**
Resume/CV: **https://tapper.codes/Resume**
Website: **https://tapper.codes**
Email: **andrew@tapper.codes**

_When I'm not coding, I can be found chasing and playing with my cat, seeing live music, and discussing philosophical thought experiments with friends over a glass of Chianti Classico._

Currently seeking new opportunities in full-stack software development - looking to work on products used worldwide.

**Comment by nikasvan** [comment]
Location: Tbilisi, Georgia

      Remote: Yes

      Willing to relocate: No

      Technologies: TypeScript, JavaScript, React / React Native / Next.js, Redux / MobX / React Query, React Testing Library, Jest, HTML / CSS

      Résumé/CV: https://cutt.ly/QLmEck3

      Email: svanidze.nikoloz@gmail.com
      
Senior Software Engineer primarily focusing on Front-end web/mobile development.

Have been working remotely for 8+ years and can collaborate productively in asynchronous teams.

Contributed to several open-source projects(React Navigation, Grafana Labs & DAO panel).

Currently searching for interesting full-time(remote) opportunities where I would be a good fit with my experience and provide the most value.

**Comment by yashwanth1796** [comment]
Location: Toronto, Canada

  Remote: Yes

  Willing to relocate: Yes

  Technologies: TypeScript, JavaScript, React / React Native / Next.js, Redux / MobX / React Query, React Testing Library, Jest, HTML / CSS

  Résumé/CV: available on request

  Email: yashwanth.vandanapu@gmail.com
I have 5+ years of experience as  Senior Software Engineer primarily focusing on Front-end web/Backend (Node JS)development.
Currently searching for interesting full-time(remote) opportunities

**Comment by adesikiru** [comment]
Hi @Nutlope, I am Ademola a 19 years old frontend developer with just 2 years of experience.
I have gone through everyone's comments and applications and I must say that I am very much intimidated by the stacks and experiences everyone here got, but I am so sorry it won't stop me from applying.

I am always ready to learn and improve myself, I promise to get better over the years and improve myself.
[Sikiru+Ademola+Resume.pdf](https://github.com/vercel/next.js/files/9439775/Sikiru%2BAdemola%2BResume.pdf).

Location: Lagos, Nigeria.
Remote: Yes.
Willing to relocate: Yes.
Technologies: HTML,CSS, SCSS, JAVASCRIPT, REACT.js, NEXT.js, BOOTSTRAP, TYPESCRIPT, TAILWIND CSS.
Resume/CV: https://docs.google.com/document/d/1zi1LwrYxX6fYuty_olpWfowUx-Jk5CX3ymGgSfshuOk/edit?usp=sharing
Website: https://github.com/ade2002
Email: sikirusulaimon2002@gmail.com

I await your response.

**Comment by Anishpras** [comment]
Hi,
My name is Anish Prashun. I have 3+ years of development with React.js and 2+ years with Next.js. I have worked on more than 25+ Projects as a freelancer. I am looking for a Fullstack / Frontend position.

Location: Bihar, India

Remote: Yes

Willing to relocate: Yes

Technologies: TypeScript, JavaScript, React, NextJS, tRPC, Prisma, GraphQL, Postman, PostgreSQL, SQL, MongoDB, NodeJS, TailwindCSS, SCSS, Redux / Recoil / Zustand, Docker.

Resume / CV : [Anish Prashun-Resume-02-09-2022.pdf](https://github.com/vercel/next.js/files/9479899/Anish.Prashun-Resume-02-09-2022.pdf)

Website: https://www.anishprashun.me/

LinkedIn: https://www.linkedin.com/in/anishpras118/

Email: anishprashun118@gmail.com

**Comment by codomposer** [comment]
Hi,
My name is Trey Duncan. I am looking for a Fullstack / Blockchain position.

Location: Thunder Bay, Canada

Remote: Yes


Technologies: TypeScript, JavaScript, React, NextJS, GraphQL, NodeJS, Tailwind CSS, SCSS, Blockchain, Solidity, Rust


Website: https://profile.codersrank.io/user/gitakileus

LinkedIn: https://www.linkedin.com/in/trey--duncan

Email: [akileus0902@gmail.com](mailto:akileus0902@gmail.com)

**Comment by kumard3** [comment]
I am a front-end developer with 3 years of React Js experience. I've worked on over 20 different projects. I've also contributed to a number of open-source projects.

Location: India
Remote: Yes (preferred)
Willing to relocate: yes
Technologies: TypeScript, JavaScript, React, NextJS, tRPC, Prisma, GraphQL, Postman, PostgreSQL, SQL, MongoDB, NodeJS, TailwindCSS, SCSS, Recoil , Docker.
Resume / CV : [Resume](https://github.com/vercel/next.js/files/9482375/kumar-deepanshu-resume.pdf)
Website: https://www.kumard3.me/
LinkedIn: https://www.linkedin.com/in/kumar-deepanshu/
Email: [kumardeepanshu157@gmail.com](mailto:kumardeepanshu157@gmail.com)

**Comment by regisrex** [comment]
Hello World!  <b>Regis</b> here.   
 <br> I'm a junior typescript developer with 2 years of experience of building projects and working in teams .
Location: Kigali
Remote: Yes (preferred)
<b>Technologies </b>: TypeScript, JavaScript, React, NextJS, Vuejs, Angular , Prisma, GraphQL, Postman, PostgreSQL, SQL, MongoDB, NodeJS, TailwindCSS, SCSS, Recoil , Nestjs.
Hire me if you're searching for a junior developer.
<b>Hourly rate </b>: 35 USD (Open for negotiations)
<b>Contact me </b> <a href="mailto:ndizihiweregis06@gmail.com">ndizihiweregis06@gmail.com</a>

**Comment by Chimmz** [comment]
Hello there.
Meet me, a Fullstack Developer (Frontend-heavy) with a total of 2 years of work experience and nearly 6 years of total programming experience. I also have a BSc. Computer Engineering.
I'm currently handling an American contract-type job to handle the development of a web application that is intended to serve about 30 million users. The development is still in progress. It is being built with Next.js, Express, Redis, and more.

Location: Nigeria
Remote: Yes
Willing to relocate: Of course, yes
Technologies: Next.js, React.js, JavaScript, TypeScript, Redux, Node.js, Express, Redis,  MongoDB, Sass, CSS, Figma
Resume/CV: https://chima-divine.vercel.app/img/my_cv-compr.pdf
Website: https://chima-divine.vercel.app/
Email: chimaorji25@gmail.com
